From 882260dce1cf719416566d3d6e0c053569c37fc9 Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Mon, 24 Feb 2020 17:18:55 +0100
Subject: [PATCH 2/5] Remove symlinks

Remove existing symlinks created by
upstream d3 in this  commit,
otherwise quilt will complain when creating
actual directories with the same name.

Upstream-Status: Inappropriate [configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 arch/arm64/boot/dts/d3 | 1 -
 drivers/d3             | 1 -
 include/d3             | 1 -
 3 files changed, 3 deletions(-)
 delete mode 120000 arch/arm64/boot/dts/d3
 delete mode 120000 drivers/d3
 delete mode 120000 include/d3

diff --git a/arch/arm64/boot/dts/d3 b/arch/arm64/boot/dts/d3
deleted file mode 120000
index 3a64c203eb95..000000000000
--- a/arch/arm64/boot/dts/d3
+++ /dev/null
@@ -1 +0,0 @@
-../../../../../../hardware/d3/
\ No newline at end of file
diff --git a/drivers/d3 b/drivers/d3
deleted file mode 120000
index 0df35689410d..000000000000
--- a/drivers/d3
+++ /dev/null
@@ -1 +0,0 @@
-../../d3/drivers/d3/
\ No newline at end of file
diff --git a/include/d3 b/include/d3
deleted file mode 120000
index 7ebaeb1ee633..000000000000
--- a/include/d3
+++ /dev/null
@@ -1 +0,0 @@
-../../d3/include/d3/
\ No newline at end of file
-- 
2.17.1

